src/components/Table/Table.module.css

```diff
@@ -1,15 +1,33 @@
+/**
```

Comment:
Why didn't the pipeline license check flag this file? 
do we check that there is the command on the pipeline that calls the script to check?

Reply:
I do not know where the pipeline instructions are, however, I noticed that the absence of the comment with the license does not result in any error 😬

Reply:
We can only run the script and see if it adds any license. Unfortunately at the moment, we are not able to check if some files need to be modified from the pipeline